frog/imports/ui/StudentView/SessionBody.jsx

```diff
@@ -17,17 +21,62 @@ const getInitialState = (activities, d = 1) => {
       };
 };
 
-const SessionBody = ({ session }: { session: Object }) => {
+const checkActivity = (activityId, operators, connections) => {
+  const connectedNodes = connections
+    .filter(x => x.target.id === activityId)
+    .map(x => x.source.id);
+  const controlOp = operators.find(x => connectedNodes.includes(x._id));
```

Comment:
This `find` method returns only one control operator if several exist. The code works well, but it means that if several control operators control one activity, only one of the operator will be used. I believed there should never be two control operators for the same activity, but the graph checks green. => Create issue 

![screenshot from 2017-09-04 08-58-06](https://user-images.githubusercontent.com/8818081/30015040-4d320dfe-914f-11e7-806c-3cec1b235501.png)